### PR TITLE
Break up global_ens precip spatial plots job

### PR DESCRIPTION
**Pull request by GwenChen-NOAA**
## Description of Changes

This PR breaks the global_ens precip spatial plots job into two smaller jobs that run under 15 min, so restart capability is not needed. The first job will generate precip maps for gefs, cmce, and ecme. The second job will generate precip maps for naefs. This PR also reduces the warning messages to debug, so they won't appear in the log files.

## Developer Questions and Checklist
* Is this a high priorty PR? If so, why and is there a date it needs to be merged by? Yes.
* Do you have any planned upcoming annual leave/PTO? No.
* Are there any changes needed for when the jobs are supposed to run? One new job is added by this PR.
  
- [X] The code changes follow [NCO's EE2 Standards](https://www.nco.ncep.noaa.gov/idsb/implementation_standards/ImplementationStandards.v11.0.0.pdf).
- [X] Developer's name is removed throughout the code and have used `${USER}` where necessary throughout the code.
- [X] References the feature branch for `HOMEevs` are removed from the code.
- [X] J-Job environment variables, COMIN and COMOUT directories, and output follow what has been [defined](https://docs.google.com/document/d/1JWg_4q80aYmmAoD21GFjp9R9y5-3w7WGM3-0HJk0Pjs/edit#heading=h.7ysbr191vzu4) for EVS.
- [X] Jobs over 15 minutes in runtime have restart capability.
- [X] If applicable, changes in the `dev/drivers/scripts` or `dev/modulefiles` have been made in the corresponding `ecf/scripts` and `ecf/defs/evs-nco.def`? 
- [X] Jobs contain the approriate file checking and don't run METplus for any missing data.
- [X] Code is using METplus wrappers structure and not calling MET executables directly.
- [X] Log is free of any ERRORs or WARNINGs.

## Testing Instructions
(1) Set up jobs

symlink the EVS_fix directory locally as "fix"

In the plots driver scripts, edit the following environment variables:

HOMEevs - set to your test EVS directory
COMIN - set to /lfs/h2/emc/vpppg/noscrub/emc.vpppg/${NET}_beta5/$evs_ver_2d

(2) Run jobs
Run the following jobs in EVS/dev/drivers/scripts/plots/global_ens:

qsub jevs_global_ens_atmos_gefs_precip_spatial_plots.sh
qsub jevs_global_ens_atmos_naefs_precip_spatial_plots.sh

A new evs.plots.global_ens.atmos.naefs.precip_spatial.vYYYYMMDD.tar file should appear containing NAEFS precip maps, and the evs.plots.global_ens.atmos.gefs.precip_spatial.vYYYYMMDD.tar file should only contains precip maps for GEFS, CMCE, and ECME.

### CODE REVIEW

**Comment by malloryprow** [comment]
Thanks for the PR!

Can you add the new ecf script in ecf/defs/evs-nco.def?

File: dev/drivers/scripts/plots/global_ens/jevs_global_ens_atmos_naefs_precip_spatial_plots.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#PBS -N jevs_global_ens_atmos_naefs_precip_spatial_plots
+#PBS -j oe 
+#PBS -S /bin/bash
+#PBS -q dev
+#PBS -A VERF-DEV
+#PBS -l walltime=00:30:00
```
Comment (malloryprow) [inline dev/drivers/scripts/plots/global_ens/jevs_global_ens_atmos_naefs_precip_spatial_plots.sh]: The ecf script has `#PBS -l walltime=00:25:00`. Can you update one of them so they match?

Comment (GwenChen-NOAA) [reply dev/drivers/scripts/plots/global_ens/jevs_global_ens_atmos_naefs_precip_spatial_plots.sh]: Just did.

**Comment by malloryprow** [comment]
Morning! I'm going to kick off the jobs to test at 1400Z, which is the time they are normally set to run.

**Comment by malloryprow** [comment]
### jevs_global_ens_atmos_gefs_precip_spatial_plots.sh
**Log File:** /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr591/EVS/dev/drivers/scripts/plots/global_ens/jevs_global_ens_atmos_gefs_precip_spatial_plots.o158877218
**COMOUT:** /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr591/evs/v2.0/plots/global_ens/atmos.20241019
**DATA:** /lfs/h2/emc/stmp/mallory.row/evs_test/prod/tmp/jevs_global_ens_atmos_gefs_precip_spatial_plots.158877218.cbqs01

### jevs_global_ens_atmos_naefs_precip_spatial_plots.sh
**Log File:** /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr591/EVS/dev/drivers/scripts/plots/global_ens/jevs_global_ens_atmos_naefs_precip_spatial_plots.o158877638
**COMOUT:** /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr591/evs/v2.0/plots/global_ens/atmos.20241019
**DATA:** /lfs/h2/emc/stmp/mallory.row/evs_test/prod/tmp/jevs_global_ens_atmos_naefs_precip_spatial_plots.158877638.cbqs01

**Comment by malloryprow** [comment]
I will note that jevs_global_ens_atmos_gefs_precip_spatial_plots.sh took longer than 15 minutes, NCO's time limit for not having restart. `resources_used.walltime = 00:15:05`

**Comment by GwenChen-NOAA** [comment]
> I will note that jevs_global_ens_atmos_gefs_precip_spatial_plots.sh took longer than 15 minutes, NCO's time limit for not having restart. `resources_used.walltime = 00:15:05`

Walltime is different from runtime and can vary from day to day (yesterday was 14 min).  I believe 15 min threshold for restart is applied to runtime. `resources_used.cput = 00:13:26`

**Comment by malloryprow** [comment]
Something to keep an eye on. If we find it is consistently going over 15 minutes jevs_global_ens_atmos_gefs_precip_spatial_plots.sh will need restart capabilities.

**Comment by GwenChen-NOAA** [comment]
> ### jevs_global_ens_atmos_gefs_precip_spatial_plots.sh
> **Log File:** /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr591/EVS/dev/drivers/scripts/plots/global_ens/jevs_global_ens_atmos_gefs_precip_spatial_plots.o158877218 **COMOUT:** /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr591/evs/v2.0/plots/global_ens/atmos.20241019 **DATA:** /lfs/h2/emc/stmp/mallory.row/evs_test/prod/tmp/jevs_global_ens_atmos_gefs_precip_spatial_plots.158877218.cbqs01
> 
> ### jevs_global_ens_atmos_naefs_precip_spatial_plots.sh
> **Log File:** /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr591/EVS/dev/drivers/scripts/plots/global_ens/jevs_global_ens_atmos_naefs_precip_spatial_plots.o158877638 **COMOUT:** /lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr591/evs/v2.0/plots/global_ens/atmos.20241019 **DATA:** /lfs/h2/emc/stmp/mallory.row/evs_test/prod/tmp/jevs_global_ens_atmos_naefs_precip_spatial_plots.158877638.cbqs01

The outputs look good to me.

**Review by malloryprow — APPROVED** [review]
Code changes are good and testing successful.

**Comment by malloryprow** [comment]
@AndrewBenjamin-NOAA This needs your review (and pending approval) when you get a chance.

**Review by AndrewBenjamin-NOAA — APPROVED** [review]
I have reviewed these changes, and the testing was successful. I am accepting the changes for this PR.

**Pull request merged**